
---
geometry_msgs/PoseStamped object_centroid
std_msgs/String object_class